lib/yarp.rb

```diff
@@ -311,6 +311,115 @@ def self.load(source, serialized)
   # This module is used for testing and debugging and is not meant to be used by
   # consumers of this library.
   module Debug
+    # For the given source, compiles with CRuby and returns a list of all of the
+    # sets of local variables that were encountered.
+    def self.cruby_locals(source)
+      verbose = $VERBOSE
+      $VERBOSE = nil
+
+      begin
+        locals = []
+        stack = [RubyVM::InstructionSequence.compile(source).to_a]
+
+        while (iseq = stack.pop)
+          if iseq[0] != :once
```

Comment:
Can we add a small top level comment around describing the layout of the iseq so `iseq[0]`, `iseq[10]` and `iseq[13]` seem less random?

Reply:
Yup!